src/ResourceManager/Resources/Commands.Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -237,6 +237,8 @@ public override void ExecuteCmdlet()
                 ExcludeAssignmentsForDeletedPrincipals = true
             };
 
+            AuthorizationClient.ValidateScope(options.Scope);
```

Comment:
options.Scope could be different than the input scope.

---
In reply to: [125784541](https://github.com/Azure/azure-powershell/pull/4228#discussion_r125784541) [](ancestors = 125784541)